modules/tinymce/src/themes/silver/main/ts/ui/statusbar/ElementPath.ts

```diff
@@ -14,81 +14,79 @@ import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
 import * as ReadOnly from '../../ReadOnly';
 import { DisablingConfigs } from '../alien/DisablingConfigs';
 
-const isHidden = (elm) => {
-  if (elm.nodeType === 1) {
-    if (elm.nodeName === 'BR' || !!elm.getAttribute('data-mce-bogus')) {
-      return true;
-    }
+interface PathData {
+  readonly name: string;
+  readonly element: Node;
+}
 
-    if (elm.getAttribute('data-mce-type') === 'bookmark') {
-      return true;
-    }
-  }
+interface ElementPathSettings {
+  readonly delimiter?: string;
+}
 
-  return false;
+const isHidden = (elm: Element): boolean => {
+  if (elm.nodeName === 'BR' || !!elm.getAttribute('data-mce-bogus')) {
+    return true;
+  } else {
+    return elm.getAttribute('data-mce-type') === 'bookmark';
+  }
 };
 
-const renderElementPath = (editor: Editor, settings, providersBackstage: UiFactoryBackstageProviders) => {
-  if (!settings.delimiter) {
-    settings.delimiter = '\u203A';
-  }
+const renderElementPath = (editor: Editor, settings: ElementPathSettings, providersBackstage: UiFactoryBackstageProviders): SimpleSpec => {
+  const delimiter = settings.delimiter ?? '\u203A';
 
-  const getDataPath = (data) => {
-    const parts = data || [];
-
-    const newPathElements = Arr.map(parts, (part, index) => Button.sketch({
-      dom: {
-        tag: 'div',
-        classes: [ 'tox-statusbar__path-item' ],
-        attributes: {
-          'role': 'button',
-          'data-index': index,
-          'tab-index': -1,
```

Comment:
This wasn't a valid attribute, so it was removed. I'm guessing it was meant to be `tabindex` but that's added by the button sketcher anyways.